st2actions/st2actions/query/mistral/v2.py

```diff
@@ -129,8 +129,8 @@ def _format_task_result(self, task):
 
         result['id'] = task['id']
         result['name'] = task['name']
-        result['execution_id'] = task.get('execution_id', None)
-        result['workflow'] = task['wf_name']
+        result['workflow_execution_id'] = task.get('workflow_execution_id', None)
```

Comment:
@m4dcoder Sounds good - it sounds like we will also need to update v0.8.0 mistral branch, right (if this hasn't been done yet)?